### PR TITLE
Daxoft/oak d sr test

### PR DESCRIPTION
**Pull request by daxoft**
add support for OAK-D-SR

### CODE REVIEW

**Comment by themarpe** [comment]
Does this library version support all the other tests as well? @daxoft

**Comment by daxoft** [comment]
> Does this library version support all the other tests as well? @daxoft

yes, it does, tested it with OAK-D-S2, Lite, Pro-PoE. 
cc @alex-luxonis to confirm

File: batch/eeprom/DM2080_R0M0E0_oak_d_sr.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "batchName": "Maxwell",
+    "batchTime": 0,
+    "boardConf": "nIR-C00M00-00",
+    "boardName": "DM2080",
+    "boardRev": "R0M0E0",
+    "productName": "OAK-D-SR",
```
Comment (themarpe) [inline batch/eeprom/DM2080_R0M0E0_oak_d_sr.json:7]: Product name: OAK-D SR

File: batch/oak_d_sr.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+    "title": "OAK-D SR",
+    "description": "OAK-D SR",
+    "image": "images/oak_d_sr.jpg",
+    "test_type": "OAK-D-SR",
+    "variants": [
+        {
+            "title":"OAK-D SR (DM2080 R0M0E0)",
```
Comment (themarpe) [inline batch/oak_d_sr.json:8]:
```suggestion
            "title":"OAK-D SR (DM2080 R0M0E0) (2x OV9782)",
```

**Review by themarpe — CHANGES_REQUESTED** [review]

**Review by themarpe — APPROVED** [review]

**Pull request merged**